.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "gremsy_base/src/gSDK_Linux"]
 	path = gremsy_base/src/gSDK_Linux
 	url = ../../Gremsy/gSDK.git
+[submodule "gremsy_base/src/gSDK"]
+	path = gremsy_base/src/gSDK
+	url = https://github.com/Gremsy/gSDK.git
```

Comment:
Please use the `../../` instead of `https://github.com/`as it better compatible with repos that are cloned via ssh.

Reply:
No problem, I'll change that.